desktop/core/src/desktop/models.py

```diff
@@ -1194,7 +1194,10 @@ class Document2(models.Model):
       db_index=True,
       verbose_name=_t('If managed under the cover by Hue and never by the user')
   )
-  is_trashed = models.NullBooleanField(default=False, db_index=True, verbose_name=_t('True if trashed'))
+  if sys.version_info[0] > 2:
```

Comment:
This is super dirty, why can't we migrate to the BooleanField with null=True?

Reply:
Django 1.11 does not support null=true -> https://docs.djangoproject.com/en/2.1/ref/models/fields/#booleanfield